code/modules/surgery/bodyparts/parts.dm

```diff
@@ -0,0 +1,307 @@
+	stam_damage_coeff = 1
+	max_stamina_damage = 120
+	var/obj/item/cavity_item
+	specific_locations = list("upper chest", "lower abdomen", "midsection", "collarbone", "lower back")
+	wound_resistance = 10
+
+/obj/item/bodypart/chest/can_dismember(obj/item/I)
+	if(!((owner.stat == DEAD) || owner.InFullCritical()))
+		return FALSE
+	return ..()
+
+/obj/item/bodypart/chest/Destroy()
+	QDEL_NULL(cavity_item)
+	return ..()
+
+/obj/item/bodypart/chest/drop_organs(mob/user, violent_removal)
+	if(cavity_item)
+		cavity_item.forceMove(drop_location())
+		cavity_item = null
+	..()
+
+/obj/item/bodypart/chest/monkey
+	icon = 'icons/mob/animal_parts.dmi'
+	icon_state = "default_monkey_chest"
+	animal_origin = MONKEY_BODYPART
+	wound_resistance = -10
+
+/obj/item/bodypart/chest/alien
+	icon = 'icons/mob/animal_parts.dmi'
+	icon_state = "alien_chest"
+	dismemberable = 0
+	max_damage = 500
+	animal_origin = ALIEN_BODYPART
+
+/obj/item/bodypart/chest/devil
+	dismemberable = 0
+	max_damage = 5000
+	animal_origin = DEVIL_BODYPART
+
+/obj/item/bodypart/chest/larva
+	icon = 'icons/mob/animal_parts.dmi'
+	icon_state = "larva_chest"
+	dismemberable = 0
+	max_damage = 50
+	animal_origin = LARVA_BODYPART
+
+/obj/item/bodypart/l_arm
+	name = "left arm"
+	desc = "Did you know that the word 'sinister' stems originally from the \
+		Latin 'sinestra' (left hand), because the left hand was supposed to \
+		be possessed by the devil? This arm appears to be possessed by no \
+		one though."
+	icon_state = "default_human_l_arm"
+	attack_verb = list("slapped", "punched")
+	max_damage = 50
+	max_stamina_damage = 50
+	body_zone = BODY_ZONE_L_ARM
+	body_part = ARM_LEFT
+	aux_zone = BODY_ZONE_PRECISE_L_HAND
+	aux_layer = HANDS_PART_LAYER
+	body_damage_coeff = 0.75
+	held_index = 1
+	px_x = -6
+	px_y = 0
+	specific_locations = list("outer left forearm", "inner left wrist", "left elbow", "left bicep", "left shoulder")
+
+/obj/item/bodypart/l_arm/is_disabled()
+	if(HAS_TRAIT(owner, TRAIT_PARALYSIS_L_ARM))
+		return BODYPART_DISABLED_PARALYSIS
+	return ..()
+
+/obj/item/bodypart/l_arm/set_disabled(new_disabled)
+	. = ..()
+	if(!.)
+		return
+	if(disabled == BODYPART_DISABLED_DAMAGE)
+		if(owner.stat < UNCONSCIOUS)
+			owner.emote("scream")
+		if(owner.stat < DEAD)
+			to_chat(owner, "<span class='userdanger'>Your [name] is too damaged to function!</span>")
+		if(held_index)
+			owner.dropItemToGround(owner.get_item_for_held_index(held_index))
+	else if(disabled == BODYPART_DISABLED_PARALYSIS)
+		if(owner.stat < DEAD)
+			to_chat(owner, "<span class='userdanger'>You can't feel your [name]!</span>")
+			if(held_index)
+				owner.dropItemToGround(owner.get_item_for_held_index(held_index))
+	if(owner.hud_used)
+		var/obj/screen/inventory/hand/L = owner.hud_used.hand_slots["[held_index]"]
+		if(L)
+			L.update_icon()
+
+/obj/item/bodypart/l_arm/monkey
+	icon = 'icons/mob/animal_parts.dmi'
+	icon_state = "default_monkey_l_arm"
+	animal_origin = MONKEY_BODYPART
+	wound_resistance = -10
```

Comment:
is it intentional that monkey heads don't get this negative wound resistance?

Reply:
normal heads have 10 wound resist anyway so i'll pay it forward to monkey heads to not be -10